indexer/ethereumetl/streaming/item_exporter_creator.py

```diff
@@ -122,6 +135,8 @@ def determine_item_exporter_type(output):
         return ItemExporterType.POSTGRES
     elif output is not None and output.startswith('gs://'):
         return ItemExporterType.GCS
+    elif output is not None and output.startswith('clickhouse'):
```

Comment:
please more precise: with protocol is using, because clickhouse have many protocol https://clickhouse.com/docs/en/interfaces/overview

Reply:
It only use to detect export target